XHR.js

```diff
@@ -0,0 +1,30 @@
+var CarLot = (function (){
+	var inventory = [];
+
+	return {
+		getInventory: function () {
+			return inventory;
+		},
+
+		setInventory: function () {
+
+			var dataRequest = new XMLHttpRequest();
+			dataRequest.addEventListener("load", function() {
+				var inventoryData = JSON.parse(this.responseText);
+				var inventoryArray = inventoryData.cars;
+				for (var i = 0; i < inventoryArray.length; i++) {
+					inventory.push(inventoryArray[i]);
+				}
+			populateDOM(inventory);
+			});
+			dataRequest.open("GET", "inventory.json");
+			dataRequest.send();
+		},
+
+		setNewDescription: function(id, userInput) {
```

Comment:
Very cool that you're changing the value in the stored array and not just on the DOM.